Sources/FoundationEssentials/CMakeLists.txt

```diff
@@ -96,7 +96,7 @@ if(NOT BUILD_SHARED_LIBS)
     target_compile_options(FoundationEssentials PRIVATE
         "SHELL:$<$<COMPILE_LANGUAGE:Swift>:-Xfrontend -public-autolink-library -Xfrontend _FoundationCShims>")
     target_compile_options(FoundationEssentials PRIVATE
-        "SHELL:$<$<COMPILE_LANGUAGE:Swift>:-Xfrontend -public-autolink-library -Xfrontend _FoundationCollections>")
+        "SHELL:$<$<COMPILE_LANGUAGE:Swift>:-Xfrontend -public-autolink-library -Xfrontend $<$<PLATFORM_ID:Windows>:${CMAKE_STATIC_LIBRARY_PREFIX_Swift}>_FoundationCollections>")
```

Comment:
Two quick questions about these variables:

1. Where is `CMAKE_STATIC_LIBRARY_PREFIX_Swift` set? Is that a known variable by CMake, or is that set somewhere else in the build script (in which case it might need a default set here for the standalone CMake build)?
2. How does `PLATFORM_ID:Windows` compare to the other "windows platform" checks we have elsewhere in CMake via `CMAKE_HOST_SYSTEM_NAME STREQUAL Windows` or `CMAKE_SYSTEM_NAME STREQUAL Windows`?

Reply:
1. It is expected to be set by the user. It will eventually be defaulted to `lib`, but the current default is `""`. The variable is known by CMake, (`CMAKE_*` variables are meant to be CMake known variables). The current default is correct until further compiler changes are merged, but this allows testing to ensure that we cover all the cases with those changes.
2. `$<PLATFORM_ID:Windows>` is a generator expression that is basically checking `CMAKE_SYSTEM_NAME STREQUAL Windows` in concept - are you building code meant to run on Windows? `CMAKE_HOST_SYSTEM_NAME STREQUAL Windows` is different in that it checks if you are building on Windows rather than building code that will run on Windows.

Reply:
Thanks for the explanation, makes sense! One more thing that I just realized: Should `_FoundationCShims` be treated the same way? It's also a static library that's statically linked into FoundationEssentials (and into FoundationInternationalization for that matter)